### ISSUE TITLE
소수의 제곱인경우 줄바꿈 나오지 않습니다 

### ISSUE THREAD

**Issue by kim-jh** [JIIIIIND/42_factorization#1]
어떤수가 소인수분해 시 가장큰소수의 2제곱 이상인 경우 줄바꿈이 나오지 않습니다 
예 18: 2*(3^2)
factorization.c 코드에서 80~81번째줄의 반복으로 인해 변수num이 1이된경우 줄바꿈이 없기때문인거 같습니다

**Comment by JIIIIIND** [comment]
맞아요. 저도 12시가 지나고 나서 갑자기 생각났었던 문제점이었죠. 간단하게 출력 부분을 연산부분 뒤로 옮기기만 하면 해결 할 수 있었을텐데 왜 이런 간단한 부분을 생각하지 못했는지 모르겠네요ㅎㅎ
